bigquery/samples/tests/conftest.py

```diff
@@ -25,14 +25,23 @@ def client():
     return bigquery.Client()
 
 
+@pytest.fixture
+def random_table_id(client, dataset_id):
+    now = datetime.datetime.now()
+    random_table_id = "example_table_{}_{}".format(
+        now.strftime("%Y%m%d%H%M%S"), uuid.uuid4().hex[:8]
+    )
+    return "{}.{}".format(dataset_id, random_table_id)
+
+
 @pytest.fixture
 def dataset_id(client):
     now = datetime.datetime.now()
     dataset_id = "python_samples_{}_{}".format(
         now.strftime("%Y%m%d%H%M%S"), uuid.uuid4().hex[:8]
     )
     dataset = client.create_dataset(dataset_id)
-    yield "{}.{}".format(dataset.project, dataset.dataset_id)
+    return "{}.{}".format(dataset.project, dataset.dataset_id)
```

Comment:
Sorry, this should stay `yield`, because it needs to clean up the dataset after the test runs.

Reply:
Fixed!